decidim-conferences/config/locales/en.yml

```
@@ -193,7 +193,7 @@ en:
           media_links_title: Media Links
         new:
           create: Create
-          title: Media Link
+          title: Create media link.
```

We should drop the dot at the end of the title as it isn't happening in other titles (and the ones that are happening we should delete them too, as in line 190 "Update media link", 273 ("Update partner."), 295 ("Update registration type."), etc) 

```suggestion
          title: Create media link
```